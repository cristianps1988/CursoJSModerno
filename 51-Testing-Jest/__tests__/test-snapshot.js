const cliente = {
    nombre: 'Cristian',
    saldo: 500,
    tipo: 'Premium'
}

describe('Verificar el cliente', () => {
    test('Verificar el cliente completo',() => {
        expect(cliente).toMatchSnapshot() // me crea una carpeta con un snapshot del cliente original, para luego compararlo
    })
})

// si quiero acutalizar el snapshot, en consola debo escribir: npm t -- -u
// tambien se puede eliminar la carpeta del snapshot