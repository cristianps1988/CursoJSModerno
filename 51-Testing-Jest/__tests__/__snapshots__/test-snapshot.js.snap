// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Verificar el cliente Verificar el cliente completo 1`] = `
{
  "nombre": "Cristian",
  "saldo": 500,
  "tipo": "Premium",
}
`;
